.github/workflows/daily.yml

```diff
@@ -1,30 +1,22 @@
 name: Daily runs
 on:
+  push:
+    paths:
+    - './.github/workflows/daily.yml'
   schedule:
     - cron: '0 5 * * *'
 
 jobs:
   linux-daily-unittests:
-    # specify PANDAS_VERSION to differentiate between nightlies and normal dependencies
-    name: "Linux - daily unit tests - Python ${{ matrix.PYTHON_VERSION }} ${{ matrix.PANDAS_VERSION }}"
+    name: "Linux - daily unit tests - Python 3.9 ${{ matrix.RUN_VERSION }}"
     runs-on: ubuntu-latest
     env:
       CI: True
     strategy:
       fail-fast: false
       matrix:
-        PYTHON_VERSION: ['3.7', '3.8']
-        include:
```

Comment:
Can you please keep the structure where we specify the version for each of the relevant packages here? That's in line with what we do elsewhere and it makes it easier for us to pin specific packages down the road.